src/SingleCellAnnotator.imjoy.html

```diff
@@ -441,6 +443,12 @@
         imageData.label_display_mode = 'multiple'
         imageData.location = response.location;
       }
+      // temporary use
+      if (this.current_task_name==='segmentation'){
+        imageData.location = false
```

Comment:
A minor comment, I would use `null` instead of `false`, they will work the same, but has different meaning.